deps/build.jl

```diff
@@ -23,6 +23,9 @@ end
 
 jupyter = get(ENV, "JUPYTER", isfile("JUPYTER") ? readchomp("JUPYTER") : is_linux() ? "jupyter" : "")
 jupyter_vers = isempty(jupyter) ? v"0.0" : prog_version(jupyter)
+if (jupyter_vers == v"0.0") && is_linux()                                       # some Linux distributions (Debian) use jupyter-notebook to launch Jupyter
```

Comment:
I would get rid of the `is_linux()` test, since presumably jupyter may be installed as `jupyter-notebook` on other systems too.